src/frontend_analysis/IR_analysis/call_graph_computation.cpp

```diff
@@ -133,10 +133,6 @@ DesignFlowStep_Status call_graph_computation::Exec()
                         "---Root function " + STR(GET_INDEX_CONST_NODE(fnode)) + " - " + symbol);
          functions.insert(GET_INDEX_CONST_NODE(fnode));
       }
-      else
-      {
-         THROW_ERROR("Function " + symbol + " not found in IR");
```

Comment:
I got the point, but I would still like to have a check to ensure the top function name specified by the user is there in the input IR.

Reply:
You are right, I will move the check before *HLSMgr->module_arch will be considered.